src/screens/Profile/Home/index.tsx

```diff
@@ -175,17 +191,48 @@ export default function ProfileHome({navigation}: ProfileScreenProps) {
             />
           </Sections.Section>
         )}
+
+        <View style={style.debugInfoContainer}>
+          {config?.installId && (
+            <View>
+              {fadeIsAnimating && (
+                <ClipboardFadeContainer>
+                  <ScreenReaderAnnouncement
+                    message={t(ProfileTexts.installId.wasCopiedAlert)}
+                  />
+                  <ThemeText>
+                    ✅ {t(ProfileTexts.installId.wasCopiedAlert)}
+                  </ThemeText>
+                </ClipboardFadeContainer>
+              )}
+              <ThemeText
+                accessibilityHint={t(ProfileTexts.installId.a11yHint)}
+                style={style.debugInfo}
+                onPress={copyInstallId}
+              >
+                {t(ProfileTexts.installId.label)}: {config.installId}
+              </ThemeText>
+            </View>
+          )}
+        </View>
       </ScrollView>
     </View>
   );
 }
 
 const useProfileHomeStyle = StyleSheet.createThemeHook((theme: Theme) => ({
   container: {
-    backgroundColor: theme.background.level1,
+    backgroundColor: theme.colors.background_1.backgroundColor,
     flex: 1,
   },
   scrollView: {
     marginTop: theme.spacings.medium,
   },
+  debugInfoContainer: {
+    alignItems: 'center',
+    marginVertical: theme.spacings.medium,
+  },
+  debugInfo: {
+    color: theme.colors.background_1.color,
+  },
```

Comment:
I think this is a workaround due to an issue with dark mode. Should not be necessary if merging in master I think.